concourse/pipelines/create-deployer.yml

```diff
@@ -467,6 +467,10 @@ jobs:
                               ./paas-cf/manifests/concourse-manifest/extensions/datadog-agent.yml
                               ./paas-cf/manifests/shared/deployments/datadog-agent.yml"
             fi
+            # shellcheck disable=SC2050
```

Comment:
(Although I dislike it) We are keeping [these kind of settings centralised in the Makefile](https://github.com/alphagov/paas-cf/blob/6e80e34acc555b151ec3ed836dfd07527a5bc286/Makefile#L135-L148), shouldn't this value be set there? 

Otherwise, we can create a .yml for environment specific configuration, and merge the value with spruce depending the environment, as we do with the [CF manifest](https://github.com/alphagov/paas-cf/tree/6e80e34acc555b151ec3ed836dfd07527a5bc286/manifests/cf-manifest/manifest/env-specific) in the [cf pipeline](https://github.com/alphagov/paas-cf/blob/6e80e34acc555b151ec3ed836dfd07527a5bc286/concourse/pipelines/create-bosh-cloudfoundry.yml#L1368)